### PR TITLE
Fixes explosive dismemberment not damaging limbs

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Which only really matters now that you can use organic limbs as replacements. Doesn't make sense a bomb going off would leave a perfectly intact and usable limb once it got blown off your body.

### CODE REVIEW

**Comment by lordpidey** [comment]
why is this DNM?

**Comment by KorPhaeron** [comment]
I put some other changes here I need to test further and explained in the op

File: code/modules/surgery/prosthetic_replacement.dm [hunk 1]
```diff
@@ -27,6 +27,9 @@
 		tool_body_zone = RP.body_zone
 	else if(istype(tool, /obj/item/bodypart))
 		var/obj/item/bodypart/L = tool
+		if(L.brute_dam + L.burn_dam >= L.max_damage/2)
```
Comment (phil235) [inline code/modules/surgery/prosthetic_replacement.dm]: bodyparts have a get_damage() proc so you don't have to manually add each damage type.

File: code/modules/surgery/prosthetic_replacement.dm [hunk 2]
```diff
@@ -27,6 +27,9 @@
...
+			user << "<span class='warning'>[tool] is too mangled to use as a replacement!</span>"
+			return
```
Comment (phil235) [inline code/modules/surgery/prosthetic_replacement.dm]: return -1 , otherwise the preop action is still considered a success.

**Comment by optimumtact** [comment]
Finish this please

**Comment by optimumtact** [comment]
Stale

**Comment by KorPhaeron** [comment]
you're stale

**Pull request closed without merging**